lib/src/repo.rs

```diff
@@ -1279,10 +1279,25 @@ impl MutableRepo {
         roots: Vec<CommitId>,
         new_parents_map: HashMap<CommitId, Vec<CommitId>>,
         options: &RewriteRefsOptions,
-        mut callback: impl FnMut(CommitRewriter) -> BackendResult<()>,
+        callback: impl FnMut(CommitRewriter) -> BackendResult<()>,
     ) -> BackendResult<()> {
         let descendants = self.find_descendants_for_rebase(roots)?;
-        let mut to_visit = self.order_commits_for_rebase(descendants, new_parents_map)?;
+        self.transform_commits(descendants, new_parents_map, options, callback)
+    }
+
+    /// Rewrite the given commits in reverse topological order.
+    ///
+    /// This function is similar to
+    /// [`Self::transform_descendants_with_options()`], but only rewrites the
+    /// `commits` provided, and does not rewrite their descendants.
+    pub fn transform_commits(
```

Comment:
This appears to be unused in this PR?

Reply:
I think I forgot to update the function to be used. Fixed.